keyboards/melody96/info.json

```diff
@@ -8,5 +8,8 @@
         }
+        "LAYOUT_hotswap": {
+              "layout": ["Esc", "F1", "F2", "F3", "F4", "F5", "F6", "F7", "F8", "F9", "F10", "F11", "F12", "PrtSc",  "Home",  "End",  "PgUp",  "PgDn", "Pause\nBreak"],  ["~\n`", "!\n1", "@\n2", "#\n3", "$\n4", "%\n5", "^\n6", "&\n7", "*\n8", "(\n9", ")\n0", "_\n-", "+\n=", {"w":2}, "Backspace", "Num Lock", "÷", "*", "-"],  [{"w":1.5}, "Tab", "Q", "W", "E", "R", "T", "Y", "U", "I", "O", "P", "{\n[", "}\n]", {"w":1.5}, "|\n\\", "7", "8\n↑", "9", {"h":2}, "+"],  [{"w":1.75}, "Caps Lock", "A", "S", "D", "F", "G", "H", "J", "K", "L", ":\n;", "\"\n'", {"w":2.25}, "Enter", "4\n←", "5", "6\n→"],  [{"w":2.25}, "Shift", "Z", "X", "C", "V", "B", "N", "M", "<\n, ", ">\n.", "?\n/", {"w":1.75}, "Shift", "↑", "1", "2\n↓", "3", {"h":2}, "Enter"],  [{"w":1.25}, "Ctrl", {"w":1.25}, "Win", {"w":1.25}, "Alt", {"a":7, "w":6.25}, "", "Ctl", "Alt", "Fn", "←", "↓", "→", "0",  "."]
+        }
```

Comment:
```suggestion
```

Reply:
D'oh. Sorry. Is the format documented anywhere? I want to make sure I get these right going forward.

Reply:
- [Configurator Support - QMK Firmware](https://docs.qmk.fm/#/reference_configurator_support)
- [info.json Format - QMK Firmware](https://docs.qmk.fm/#/reference_info_json)

First doc is more of a walk-through; the second is more technical documentation.

Reply:
Danke!